policies/modules/ROOT/pages/policies-automated-overview.adoc

```diff
@@ -2,22 +2,20 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:page-aliases: api-manager::automated-policies-landing-page.adoc
+// :page-aliases: api-manager::automated-policies-landing-page.adoc
 
-Configuring and using automated policies enables your Anypoint Platform instance to comply with common security and logging requirements by automatically applying the same set of policies to all APIs running in a single environment. With policy automation, API owners can quickly design, build, and deploy secure and consistent APIs.
+Use automated policies to enforce security and logging requirements by applying the same policies to all APIs running in Flex Gateway or Mule. With policy automation, you can quickly design, build, and deploy secure and consistent APIs.
 
-With automated policies, API owners can deploy their APIs with confidence because the target environment automatically enforces critical polices. This ensures consistency and reduces possible errors. An administrator no longer needs to remember to insert a policy to enforce common standards manually.
+Automated policies enable you to deploy APIs with confidence because the target environment automatically enforces critical polices, resulting in reduced errors. An administrator no longer needs to remember to insert a policy to enforce common standards manually.
```

Comment:
```suggestion
Automated policies enable you to deploy APIs with confidence because the target environment automatically enforces critical policies, resulting in reduced errors. An administrator no longer needs to remember to insert a policy to enforce common standards manually.
```